### DISCUSSION TITLE
Switching from Chlorine Tabs to Salt Cell Generator

### DISCUSSION THREAD

**Discussion by spifftek70** [tagyoureit/nodejs-poolController#993] [Nixie]
I have a 25,000 gallon pool running Nixie shared body configuration. I want to switch from chlorine tabs to a Salt Cell Generator.  What is the most affordable solution?  Can I get JUST the Salt Cell Generator or do I need the controller to interface?  Also, is there a wiring diagram?

**Comment by rstrouse** [comment]
For a 25kgal pool I would suggest a Pentair IC60 with the power supply unless you already have a power supply and surge board.   If you used to have a Pentair system you may already have one.  Wiring the cell is a no-brainer and you will simply connect the RS485 to the surge board.   I know there are a bunch of knockoffs out there but none of them provide RS485 support to manage the output.

Just a word on the rated chlorine supply from these cells.  Unfortunately, they rate them at 24hours without any loss over that time.  Typically you need a cell that produces at least double the rated amount because many do not run their chlorinator 24 hours a day in darkness.

The cell will save you from being married to the pool on top of the crazy high cost of chlorine these days.

**Reply by spifftek70** [reply]
"but none of them provide RS485 support to manage the output." - only Pentair does RS485?  If so, that's good to know.  

I don't have a Pentair "System".  I have a Pentair Inteliflow pump and UltraTemp (both RS485) that connect to my Nixie NodeJS pool Controller and Relay Control Manager. I see different variations on this device (IC60).  Can you provide a good like for what I need to make this work?  See the below diagram.  Thanks.

![pool automation-12 drawio-19-6 drawio](https://github.com/tagyoureit/nodejs-poolController/assets/11727190/b1a4fd13-dfd1-4c4d-b2d7-3528ce05d5f8)

**Reply by rstrouse** [reply]
You are looking for the IC60 with power center.  The power center will hook to either 120v or 220v depending on how you wire it through the load side of a 3hp relay assigned to the filter in dashPanel.  You want this to be connected this way so that the power to the chlorinator load center is cut when the pool or spa is not running.  On the remote chance that the flow sensor on the cell fails closed because of debris in the pipe it will not fill the pipes with hydrogen gas.

Then the RS485 bus is connected to the surge board located in the power center to control the chlorine output.  I do suggest the IC60 for your size pool.  Honestly, this is probably the best thing you can do for your pool and your own well being.

The part numbers are EC-520556 for the power center and EC-521105 for the cell.  Often you can find these bundled together.

**Reply by spifftek70** [reply]
Just to clarify, my filter is not powered, so it has no power supply.  My Pentair IntelliFlow pump that feeds the filter requires no relay as it connects RJ485 to my Raspberry Pi via the Sequent MEGA-BAS board for connectivity and directly to 220V for power.  With that in mind, do I need another 3HP 110V Relay such as the one my Blower connects to?

**Comment by rstrouse** [comment]
That is correct.  The relay however is for when the pool or spa circuit is engaged.   The only thing you will have on this relay is the chlorinator.   Do not add it to a separate circuit and name it chlorinator.  That is not the recommended or desired setup.

**Comment by spifftek70** [comment]
Okay, So this is what I need?
https://www.poolsupplyunlimited.com/pool/pentair-ic60kit-intellichlor-salt-chlorine-generator/136371p1

Does this problem with hydrogen in the pipes happen often?

**Reply by rstrouse** [reply]
Yes that is it.   While it does not happen often it happens enough that you will find news reports about it.  Typically the chain of events that make this happen are debris getting past the filter that pins the flow switch closed.  Then the cell continues to generate chlorine and the bubbles that result from the generation are hydrogen and chlorine gas.  In some instances this can even back up into the filter and when the pump finally comes on there is a big boom.

**Reply by spifftek70** [reply]
holy cow!  Good to know.

**Comment by spifftek70** [comment]
Here is the diagram of my pool/spa water flow and where I believe (from research) I would place the IC60 within that system.  After the UltraTemp, but before the output valve.  Is this correct?

![Waterflow diagram drawio](https://github.com/user-attachments/assets/27abea8f-5064-464a-9e0e-0772046d1ad5)

**Reply by tagyoureit** [reply]
If the pool or spa circulation pump will output water at >80gpm, the chlorinator should be installed with a loop.  Otherwise, looks good.

**Reply by spifftek70** [reply]
they’ll pump 50 gallons per minute at most
